config/hyperion.config.json.commented

```diff
@@ -138,7 +140,9 @@
 		"sDVOffsetMin"         : 0.25,
 		"sDHOffsetMin"         : 0.25,
 		"sDVOffsetMax"         : 0.75,
-		"sDHOffsetMax"         : 0.75
+		"sDHOffsetMax"         : 0.75,
+		"hdrToneMapping"		   : false,
+		"fpsSoftwareDecimation"	: 1		
```

Comment:
This would be more clear if we name it capture rate or capture frequency and use accordingly, like the other grabbers.